webdav3/client.py

```diff
@@ -849,6 +853,38 @@ class WebDavXmlUtils:
     def __init__(self):
         pass
 
+    @staticmethod
+    def parse_get_list_info_response(content):
```

Comment:
@huangganggui please add unit tests for that method in tests/test_client_unit.py